### DISCUSSION TITLE
TFT_eSPI with 2.4 TFT How to activate Touch????

### DISCUSSION THREAD

**Discussion by xHerrVorragendx** [Bodmer/TFT_eSPI#1180] [Q&A - Touch]
Hey Guys,
im a bit confused. I am working on it like 5 days straight, but my touch functionality won't work.
I'm using a az_delivery esp32 (https://www.az-delivery.de/products/esp32-developmentboard) and the az-delivery 2.4 tft (http://www.lcdwiki.com/2.4inch_Arduino_Display).

Display work's fine for me. But how can i use the TFT-Touch-Function?
Do i have to use another library or just a wrong user_setup.h?

BTW. here's my user_setup.h

`
#define ILI9341_DRIVER

#define TFT_PARALLEL_8_BIT

#define TFT_CS 27 // Chip select control pin
#define TFT_DC 14 // Data Command control pin - must use a pin in the range 0-31
#define TFT_RST 26 // Reset pin
 
#define TFT_WR 12 // Write strobe control pin - must use a pin in the range 0-31
#define TFT_RD 13

#define TFT_D0 16 // Must use pins in the range 0-31 for the data bus
#define TFT_D1 4 // so a single register write sets/clears all bits
#define TFT_D2 23
#define TFT_D3 22
#define TFT_D4 21
#define TFT_D5 19
#define TFT_D6 18
#define TFT_D7 17
....`

I would be soooo glad if anyone could help me out.
THX!

**Comment by Bodmer** [comment]
See last paragraph of the [ReadMe here](https://github.com/Bodmer/TFT_eSPI#8-bit-parallel-support).
